.github/workflows/cli.yml

```diff
@@ -71,7 +68,7 @@ jobs:
       working-directory: ./myworkspace
       run: |
         cd init1
-        ../../packages/cli/dist/bin/index.js run ./examples/pipelines/text-bayes-classification.json
+        ../../packages/cli/dist/bin/index.js run ../../example/pipelines/mnist-image-classification.json
```

Comment:
We still need to use text-bayes-classification for cli workflow, minist will take more time to complete.